lv_drv_conf_template.h

```diff
@@ -384,6 +384,9 @@
 
 #if USE_LIBINPUT || USE_BSD_LIBINPUT
 #  define LIBINPUT_NAME   "/dev/input/event0"        /*You can use the "evtest" Linux tool to get the list of devices and test them*/
+
+/*If only a single device of the same type is connected, you can also auto detect it, e.g.:
+ *#  define LIBINPUT_NAME   libinput_find_dev(LIBINPUT_CAPABILITY_TOUCH, false)*/
```

Comment:
Please move this comment above `#  define LIBINPUT_NAME   "/dev/input/event0"`.

Reply:
Good idea. Made the change.